CHANGES.rst

```diff
@@ -129,6 +129,8 @@ astropy.time
 
 - Added `datetime.timedelta` format class for ``TimeDelta``. [#7441]
 
+- Added `strftime` and `strptime` methods to ``Time`` class. [#7323]
```

Comment:
Both `strftime` and `strptime` should use double backticks.  Also give the readers some context for why they should care:
```
These methods are similar to those in the Python standard library 
`time` package and provide flexible input and output formatting.
```

Reply:
I do not seem to get the hang of back ticks. It should be fixed now.